src/autorest-core/lib/outstanding-task-awaiter.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { OutstandingTaskAlreadyCompletedException } from './exception';
+import { OutstandingTaskAlreadyCompletedException } from "./exception";
```

Comment:
this is not for locking (which would also indicate bad design), this is merely a convenience layer over `Promise.all`. I'm no node library expert, I'd be glad to learn about such about a library!